src/html-conventions-lib/general-connectors-html-convention.xsl

```diff
@@ -688,4 +688,133 @@
         </xsl:if>
     </xsl:template>
 
+    <xd:doc>
+        <xd:desc>[common-name-unique] - The connector role name $value$ is not unique. </xd:desc>
+        <xd:param name="connector"/>
+    </xd:doc>
+    <xsl:template name="connectorUniqueName">
+        <xsl:param name="connector"/>
+        <xsl:variable name="sourceRole" select="$connector/source/role/@name"/>
+        <xsl:variable name="targetRole" select="$connector/target/role/@name"/>
+
+        <xsl:if test="boolean($sourceRole)">
+            <xsl:variable name="elementsFound"
+                select="f:getElementByName($sourceRole, root($connector))"/>
+            <xsl:variable name="attributesFound"
+                select="f:getAttributeByName($sourceRole, root($connector))"/>
+
+            <xsl:sequence
+                select="
+                    if (count($elementsFound) > 1 or count($attributesFound) > 0) then
```

Comment:
this logic is strange. Please recheck based on similar comment above.

Reply:
fixed mistake. Yes it should >0